src/internal/components/autosuggest-input/styles.scss

```diff
@@ -8,7 +8,6 @@
 }
 
 .dropdown-footer {
-  // Fixes incorrect flex height calculation in IE11.
```

Comment:
Didn't find any reference what style property fixes this problem on IE11, decided to leave it as is